src/sagemaker_xgboost_container/algorithm_mode/serve.py

```diff
@@ -266,6 +303,15 @@ def invocations():
         logging.exception(e)
         return flask.Response(response="Unable to evaluate payload provided: %s" % e, status=http.client.BAD_REQUEST)
 
+    if serve_utils.is_selectable_inference_response():
+        try:
+            accept = _parse_accept(flask.request)
+        except Exception as e:
+            logging.exception(e)
+            return flask.Response(response=str(e), status=http.client.NOT_ACCEPTABLE)
+
+        return _handle_selectable_inference_response(preds, accept)
```

Comment:
This seems a little different from what was discussed; I thought the selectable response would only be generated based on an env var that is set. However, we are parsing for configs that might not be used regardless of env vars that is set.

Reply:
This line is only hit if `serve_utils.is_selectable_inference_response()` is `True`, which does the env var check. I may be missing something but what extraneous configs are you referring to when that env var is not set?